docs/try/from-source.md

```diff
@@ -0,0 +1,58 @@
+# Running `witnet-rust` from source code
+
+## Install compilation dependencies
+
+### Rust 
+
+`witnet-rust` is developed using Rust programming language.
+You need to have it installed on your system to be able to compile the source code.
+Follow installation instructions for your operating system provided on [rustup.rs][rustup]
+
+### Compilation dependencies
+
+```console tab="GNU/Linux (apt)"
+apt install -y clang git libssl-dev protobuf-compiler librocksdb-dev
+```
+
+```console tab="macOS"
+xcode-select --install
+
+brew install git openssl protobuf rocksdb
+```
+
+## Clone source code from Witnet GitHub repository
+
+```console tab="HTTPS"
+git clone https://github.com/witnet/witnet-rust.git
+cd witnet-rust
+```
+
+## Get latest genesis_block.json
+
+```console
+curl https://raw.githubusercontent.com/witnet/genesis_block/master/testnet_0.7.1_genesis_block.json -o genesis_block.json
```

Comment:
This other URL will always point to the latest version so that we don't need to update the documentation every time we change the genesis block:
```suggestion
curl https://raw.githubusercontent.com/witnet/genesis_block/master/latest/genesis_block.json -o genesis_block.json
```

Reply:
:+1: